lib/core/address_validator.dart

```diff
@@ -171,6 +171,7 @@ class AddressValidator extends TextValidator {
       case CryptoCurrency.shib:
       case CryptoCurrency.avaxc:
       case CryptoCurrency.bch:
+        return [42, 43, 44, 54, 55];
```

Comment:
As I understand, this will permit these non-42 lengths for all the preceding ETH assets as well, is this desired?